code/mob/living/critter/flock/flockdrone.dm

```diff
@@ -399,6 +416,8 @@
 	if (src.dormant)
 		return
 
+	src.do_antigrab()
```

Comment:
This proc is only used here, consider in-lining it? It just adds proc call overhead.